rules/index.md

```diff
@@ -1,14 +1,18 @@
 ---
 layout: page
-title: 2019 Rules
+title: 2020 Rules
 ---
 
-2019 Rules
+2020 Rules
 ==========
 
-[<img class="left" src="{{ site.baseurl }}/resources/2019/rulebook.png" />]({{ site.baseurl }}/resources/2019/rulebook.pdf)
-The rules, regulations and specifications for the SR2019 competition can be
-found in the [rulebook]({{ site.baseurl }}/resources/2019/rulebook.pdf).
+The rules for this year's competition will be published shortly after Kickstart.
```

Comment:
The current intention is to publish them _during_ kickstart rather than after (https://github.com/srobo/tasks/issues/376)

```suggestion
The rules for this year's competition will be published during Kickstart.
```

Reply:
:+1: (this was blindly copied from what it said last year)

Reply:
https://github.com/srobo/docs/pull/107/commits/d2382cca54b19803e0deecb6dd36f7068b6b6241